### ISSUE TITLE
Running MakeCode locally does not save projects on filesystem

### ISSUE THREAD

**Issue by lobodpav** [microsoft/pxt-microbit#6021]
**Describe the bug**

Following the [official instructions](https://makecode.com/cli) I have run MakeCode locally.

The documentation states that
> If you have created a PXT project from the web browser, you can go to its folder (it will sit under projects) and use the CLI to build and deploy it.

However, the `project` directory is empty.

**To Reproduce**
1. Go to a folder of your liking, say `/Users/pavel/makecode`
2. As stated in the CLI documentation, run:
    ```shell
    npm install -g pxt
    mkdir microbit
    cd microbit
    pxt target microbit
    pxt serve
    ```
3. The application opens up in Chrome at `http://localhost:3232/index.html#editor`
4. Create a new project called `test` and do some changes
5. Run `ls -la /Users/pavel/makecode/microbit/projects`

**Expected behavior**
The directory listing should show a `test` directory with the project content.

**Actual behavior**
The `/Users/pavel/makecode/microbit/projects` directory is empty.

**Desktop (please complete the following information):**
 - Computer: MacBook Pro M2 Max
 - OS: macOS Sequoia 15.1.1
 - Browser: Chrome v131.0.6778.86
 - pxt-core: v11.3.3

**Comment by tballmsft** [comment]
I don't think that this has ever been the behavior of PXT running locally from localhost. The projects are stored in browser local storage. @abchatra ?

**Comment by tballmsft** [comment]
I think this may have been true in the very distant past. Probably the docs should be updated.

**Comment by lobodpav** [comment]
Right, so what's the recommended local development approach when coding micro:bit apps?
In my eyes, developing in VS Code is a far better experience than in MakeCode.
However, the absence of a simulator means that I have to push the code to GitHub, go to MakeCode on the web and pull the changes.

**Comment by tballmsft** [comment]
Understand about local development; two things:

1. The makecode web app has github integration; have you tried that? Not local solution, of course.
2.  There is a branch of vs code for makecode with micro:bit here: https://github.com/microsoft/vscode-makecode/pull/179 - still needs some love.  Try it out and

**Comment by abchatra** [comment]
There is a flag for this. Pass "?ws=fs" to your localhost URL parameter and files should be saved in the filesystem.
